inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/SinkType.java

```diff
@@ -17,9 +17,17 @@
 
 package org.apache.inlong.manager.common.consts;
 
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
 /**
  * Constants of sink type.
  */
+@Component
```

Comment:
I mean, you can customize two annotations: one for types supporting `Sort, such as `@SupportSort`, and another for types supporting `SortStandalone`, such as `@SupportSortStandalone`.

On each Sink type, specify the supported annotation types. During the InLong-Manager startup, use reflection to read specific annotations and categorize Sink types into different `Set`.

With this approach, when adding a new Sink type, you only need to annotate it with one or both of the mentioned annotations, eliminating the need to maintain a configuration file.